docs/user-guide.md

```diff
@@ -161,35 +175,56 @@ from basemodel import BaseModel
+code.
 
 ### Supported Models
 
-Bailo currently builds model images using [seldonio's Python 3.7 s2i image](https://hub.docker.com/r/seldonio/seldon-core-s2i-python37). This means Bailo currently only supports models that can be run with Python 3.7. Generally, the packages your model uses need to be pip-installable and listed in the [requirements.txt](../examples/example_model_code/requirements.txt) file. NumPy (1.21 as of this writing) is provided, by default.
+Bailo currently builds model images using
+[seldonio's Python 3.7 s2i image](https://hub.docker.com/r/seldonio/seldon-core-s2i-python37). This means Bailo
+currently only supports models that can be run with Python 3.7. Generally, the packages your model uses need to be
+pip-installable and listed in the [requirements.txt](../examples/example_model_code/requirements.txt) file. NumPy (1.21
+as of this writing) is provided, by default.
 
 ## Requesting a deloyment
 
-Once a model has been added to the marketplace, it can be found by other users. If the model matches a user's use case, they can request a deployment of the model (by clicking "Request Deployment" on a model's page). Similar to the model upload form, there's a deployment request form to fill out (along with an option to request a deployment with the form data populated in a json string, via the "Upload Deployment" tab).
+Once a model has been added to the marketplace, it can be found by other users. If the model matches a user's use case,
+they can request a deployment of the model (by clicking "Request Deployment" on a model's page). Similar to the model
+upload form, there's a deployment request form to fill out (along with an option to request a deployment with the form
+data populated in a json string, via the "Upload Deployment" tab).
 
-Similar to the model upload form, you will need to supply a secondary POC for the deployment (someone to contact about the deployment if the deployment owner is unable to be reached).
+Similar to the model upload form, you will need to supply a secondary POC for the deployment (someone to contact about
+the deployment if the deployment owner is unable to be reached).
 
-If your request is approved by the model's manager, then a Docker image corresponding to the requested model is made available. You are then able to pull this image and run it on your own infrastructure.
+If your request is approved by the model's manager, then a Docker image corresponding to the requested model is made
+available. You are then able to pull this image and run it on your own infrastructure.
 
 ## Updating a model
 
-Updating a model is not yet implemented, however in the future models can be updated by going to the model's url, clicking the settings tab, then clicking "Upload New Version":
+Updating a model is not yet implemented, however in the future models can be updated by going to the model's url,
```

Comment:
Editing and new versions for models is now possible - although I don't mind leaving this as it is for now and updating a later point.